demosdk/pom.xml

```diff
@@ -193,6 +201,12 @@
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>5.20.0</version>
```

Comment:
declare version on top in properties as <mockito.core.version>5.20.0</mockito.core.version>